.github/workflows/CD.yml

```diff
@@ -552,11 +555,18 @@ jobs:
           username: "${{ secrets.DOCKERHUB_RELEASE_USERNAME }}"
           password: "${{ secrets.DOCKERHUB_RELEASE_TOKEN }}"
 
-      - name: Tag latest and upload to docker
+      - name: Tag version and upload to release dockerhub
+        run: |
+          TAG=`cat build/packages/VERSION`
+          REPO_NAME="$IMAGE_NAMESPACE/$IMAGE_NAME"
+          docker tag public.ecr.aws/$ECR_REPO_INTEGRATION_TEST:$TESTING_TAG $REPO_NAME:$TAG
+          docker push $REPO_NAME:$TAG
+
+      - name: Tag latest and upload to release dockerhub
         if: steps.version.outputs.any-update == 'true'
         run: |
           REPO_NAME="$IMAGE_NAMESPACE/$IMAGE_NAME"
-          docker tag $IMAGE_NAME $REPO_NAME:latest
+          docker tag public.ecr.aws/$ECR_REPO_INTEGRATION_TEST:$TESTING_TAG $REPO_NAME:latest
```

Comment:
Where are you setting this `$TESTING_TAG`?

Reply:
Fixed in it the newest commit. Set these [in the versioning step and release-version-image jobs ](https://github.com/aws-observability/aws-otel-collector/pull/860/files#diff-a64cbb6e953e697cb4302ab31af9ebbd95a8de6216f874bb0ffdefce8e3c1cd9R334-R340)